### DISCUSSION TITLE
Is it possible to use both Ollama and OpenAI

### DISCUSSION THREAD

**Discussion by OlaoluwaM** [open-webui/open-webui#3356] [Q&A]
Is there a setup that allows for using both Ollama and OpenAI models?

**Comment by HearthCore** [comment]
By Default, even at the same time in one Chat window with each result having their own options to recreate, copy, edit and prompt the llm to continue.

You can also use all other OpenAI compatible API's like Mistral or other local API Providers like LLMStudio/Server.
Or run multiple Ollama Servers on different Hardware for their specific model duties if neccesairy.
